Show a confirmation of successfully sending BTC or BSQ from wallet

jmacxx opened the pull request:
As requested, this shows a message of successful sending.
The confirmation details include amount, recipient address and txId.
Includes a link to open txId details in the user's external block explorer.
To be consistent it does the same for both BSQ and BTC.

Fixes #5067 

### Screenshots:

![SendBsqTxId](https://user-images.githubusercontent.com/47253594/104113006-04556780-52bb-11eb-824c-159942d54cee.png)

Note the technicality with sending BSQ is you cannot look up the details in a BSQ explorer until +1 bitcoin confs have passed.

@m52go any thoughts on improving the note text shown in the above popup?

https://github.com/jmacxx/bisq/blob/8f24f0ab9c6d12143c09ecd1bb3bcbaa082bb3e4/core/src/main/resources/i18n/displayStrings.properties#L2699

--- 

![SendBtcTx](https://user-images.githubusercontent.com/47253594/104113008-06b7c180-52bb-11eb-8b82-6adc718ab389.png)

m52go commented on core/src/main/resources/i18n/displayStrings.properties:
```diff
@@ -2693,6 +2693,13 @@ tradeDetailsWindow.tradeState=Trade state
+txDetailsWindow.headline=Transaction Details
+txDetailsWindow.btc.note=You have sent BTC.
+txDetailsWindow.bsq.note=You have sent BSQ funds.  \
+  Please note that BSQ is colored Bitcoin, it will not show up in a BSQ explorer until at least 1 Bitcoin block has been confirmed.
```
I got the impression this string is saying the BSQ transaction will show in the BSQ explorer as soon as 1 more bitcoin block is confirmed, whether it includes the BSQ transaction or not. 

Suggestion to make more specific:

```suggestion
txDetailsWindow.bsq.note=You have sent BSQ funds.  \
  BSQ is colored bitcoin, so the transaction will not show in a BSQ explorer until it has been confirmed in a bitcoin block.
```

wiz commented:
Nice!! Not related to this PR but it would be cool to have 2 links to the block explorer, one for clearnet and one for Tor

pazza83 commented:
Thanks for doing this so quickly. 

Would also be good if mining fees where displayed.

ghost commented:
Mining fees are displayed on the "Confirm withdrawal request" popup immediately prior.  There's no need to display them after sending.
![Screenshot from 2021-01-10 14-10-23](https://user-images.githubusercontent.com/47253594/104134189-c35a6300-534d-11eb-958a-c4304b9991bd.png)

Commit:
Show a confirmation of successfully sending BTC or BSQ from wallet

The confirmation details include amount, recipient address and txId.
Includes a link to open txId details in the user's external block explorer.
@m52go reworded the BSQ conf explanation note.
Added "don't show again" checkbox.

ghost commented:
Added "Don't show again" checkbox.

![image](https://user-images.githubusercontent.com/47253594/104141580-1f88ab80-537d-11eb-95a8-654fd9df1ee2.png)

ripcurlx commented on desktop/src/main/java/bisq/desktop/main/overlays/windows/TxDetails.java:
```diff
@@ -0,0 +1,66 @@
+    protected void addContent() {
+        GridPane.setColumnSpan(
+                addMultilineLabel(gridPane, ++rowIndex, note, 0), 2);
+        gridPane.add(new Label(""), 0, ++rowIndex);  // spacer
```
I normally use for this cases `Region` which is more lightweight as `Label`

e.g. 
```suggestion
        Region spacer = new Region();
        spacer.setMinHeight(20);
        gridPane.add(spacer, 0, ++rowIndex);
```
Would have the same visual appearance.

ripcurlx commented on desktop/src/main/java/bisq/desktop/main/dao/wallet/send/BsqSendView.java:
```diff
@@ -373,6 +375,12 @@ private void showPublishTxPopup(Coin receiverAmount,
                         @Override
                         public void onSuccess(Transaction transaction) {
                             log.debug("Successfully sent tx with id {}", txWithBtcFee.getTxId().toString());
+                            String key = "showTransactionSentBsq";
```
It would be great to get rid of the error message on the empty fields after the successful transfer as well.
For the `BsqSendView` this would mean changing in the `showPublishTxPopup` resultHandler into
```
() -> {
       receiversAddressInputTextField.setValidator(null);
       receiversAddressInputTextField.setText("");
       receiversAddressInputTextField.setValidator(bsqAddressValidator);
       amountInputTextField.setValidator(null);
       amountInputTextField.setText("");
       amountInputTextField.setValidator(bsqValidator);
       });
```

ripcurlx replied:
Not beautiful, but it works :wink:

ripcurlx reviewed (changes requested):
NACK - it does work as expected, but I would suggest two small changes to use a more lightweight container in one case and improve the UX after the success popup is closed. Thanks for picking up this issue!

Commit:
Code review suggestions from @ripcurlx

Disable validation errors when clearing edit fields
Implement spacer field using Region
